birdhouse/optional-components/canarie-api-full-monitoring/config/thredds/canarie_api_full_monitoring.py.template

```diff
@@ -0,0 +1,16 @@
+import copy
+
+bird_public = "Thredds-public"
+bird_public_cfg = {
+    'request': {
+        'url': 'https://${PAVICS_FQDN_PUBLIC}/thredds/catalog.html',
+    },
+}
+
+bird_private = bird_public.split("-public")[0]
+if bird_private in SERVICES:
+    SERVICES[bird_private]["monitoring"][bird_public] = bird_public_cfg
+elif "node" in SERVICES:
+    SERVICES["node"]["monitoring"][bird_public] = bird_public_cfg
```

Comment:
A bit odd to refer to "bird" for THREDDS. 
Looks to be from simple copy-paste. 
Could be generalized to `service_public` across all scripts to keep them uniform.

Not blocker though.

Reply:
yes it's from copy/paste. previously thredds, catalog, ncwms2 were all described as birds in this file.